### DISCUSSION TITLE
Positive Law (Legislation, Statutes, Etc.)

### DISCUSSION THREAD

**Discussion by halfprice06** [freelawproject/courtlistener#4302] [Ideas]
Has FLP ever considered adding Legislation to the Court Listener Project db? or a new db just for this? 

My dream is oneday a completely open source replacement for Westlaw. 

XD

**Comment by mlissner** [comment]
We certainly talk about it from time to time, but it's never made it off the back burner. Doing it well is pretty tough. It does feel like a necessary ingredient in our recipe though.
